src/System.IO.Pipes/tests/NativeOverlapped.unix.cs

```diff
@@ -0,0 +1,3 @@
+
+// stub implementation of NativeOverlapped to compile windows only tests in Unix build
+internal struct NativeOverlapped {}
```

Comment:
I started doing that and it was too large a change to just unblock this.  Also, we can remove this once I make the notsupported assembly for Overlapped.